net/openconnect/files/openconnect-wrapper

```diff
@@ -10,32 +10,4 @@ test -z "$1" && exit 1
 
 pwfile=$1
 shift
-
-pidfile=/var/run/ocwrap-$$.pid
-
-cleanup()
-{
-	if ! test -z "$pid";then
-		kill $pid
-		wait $pid
-	fi
-	exit 0
-}
-
-cleanup2()
-{
-	if ! test -z "$pid";then
-		kill -2 $pid
-		wait $pid
-	fi
-	exit 0
-}
-
-trap cleanup2 2
-trap cleanup 1 3 6 15
```

Comment:
If there are no other routes other than those reported to netifd by vpnc-script with `proto_add_ipv4_route`, then netifd should take care of removing those routes on link down.

We also have `proto_kill_command` "$config" 2 in `proto_openconnect_teardown`.  This should also suffice